Adding information layer (#157)
International-Data-Spaces-Association/IDS-RAM_4_0: documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md

Model. As such, it is accompanied by a set of auxiliary resources (e.g.
guidance documents, reference examples, validation tools, and editing
tools intended to support a competent, appropriate, and consistent usage
of the IDS Vocabulary).

#### Conceptual Representation

The Conceptual Representation of the Information Model presents a
high-level overview of the main, largely invariant concepts, with no
commitment to a particular technology or domain. It targets a general
audience, management boards, and media, as it provides basic information
and promotes a shared understanding of the concepts by means of a
textual document and a plausible visual notation. If available,
references to related elements of the Declarative Representation and a
Programmatic Representation are provided, encouraging the reader to take
a look at these alternative implementations.

#### Declarative Representation

The Declarative Representation (IDS Vocabulary) provides a normative
view of the Information Model of the International Data Spaces.[^4] It
has been developed along the analysis, findings, and requirements of the
Conceptual Representation. Based on a stack of W3C Semantic Web

> technology standards[^5] and standard modeling vocabularies (DCAT[^6],
> ODRL[^7], SKOS[^8], etc.), it provides a formal, machine-interpretable

Reviewer: ```suggestion
technology standards[^5] and standard modeling vocabularies (the Data Catalog Vocabulary DCAT[^6],
the Open Digital Rights Language ODRL[^7], the Simple Knowledge Organization System SKOS[^8], etc.), it provides a formal, machine-interpretable
```
expanded the standards acronyms

specification of concepts envisaged by the Conceptual Representation,

> residing at the persistent namespace URI <https://w3id.org/idsa/core/>.

Reviewer: ```suggestion
residing at the persistent namespace URI <https://w3id.org/idsa/core/> according to best practices for publishing linked data[^bp-ld].
```

Furthermore, it details and formally defines entities of the
International Data Spaces in order to be able to share, search for, and

> reason upon the structured metadata describing these entities. As such,
> it comprises a complete referential model allowing the derivation of a

Reviewer: ```suggestion
reason upon the structured metadata describing these entities. The IDS Vocabulary is defined using RDF Schema[^rdfs] and the OWL Web Ontology Language[^owl]; additionally, descriptions of Digital Resources can be validated against SHACL shapes[^shacl] that express syntactic and semantic conditions. Queries against, e.g., Data Resources listed in the Data Catalogue of a Connector or Broker, or against Software Resources available from an App Store, can be formulated in query languages such as SPARQL[^sparql].  Thus, the
Declarative Representation comprises a complete referential model allowing the derivation of a
```
Now also explicitly referencing the basic W3C standards mentioned in the figure.

clarity and consistency. Similar to the principle a microscope works,
each concern follows a particular, analytical point of view, while other
concerns can temporarily be disregarded. This principle can be applied
to information modeling, aiming at a thorough understanding of the
domain and fostering modularity and re-usability of the resulting (sub-)
models. Accordingly designed, these models may evolve independently of
each other and can be updated by different agents at different times. As
any modification of a single element of the overall model does not
require a change in other, logically unrelated parts, the development
and maintenance of models can be substantially simplified.

::: Center
![Outline of the Concern-Basic concern
hexagon](./media/image32.png){#fig:Outline_of_the_ConcernBasic_concern_hexagon
width="3.64in" height="2.67in"}
:::

#### Concern Hexagon

To illustrate the main modeling [c]{.underline}oncerns of Digital
Resources in a way easy to memorize, the mnemonic hexagonal arrangement
of [c]{.underline}arbon atoms can be used (C-Hexagon), as shown in
Figure
[2](#fig:Outline_of_the_ConcernBasic_concern_hexagon){reference-type="ref"

> reference="fig:Outline_of_the_ConcernBasic_concern_hexagon"}. As a

Reviewer: ```suggestion
reference="fig:Outline_of_the_ConcernBasic_concern_hexagon"}.

As a
```
Now spending one paragraph on each concern, reintroducing revised versions of the summaries we had in the RAM 3.0.

Resource's content is its most essential aspect, Content is located at

> the top of the hexagon. This content is interpretable by references to a

Reviewer: ```suggestion
the top of the hexagon. The *Content* concern deals with 
1. the description of a Resource's abstract substance, 
2. its serialization as a representation in a machine-interpretable format, and 
3. the materializations of these representations at certain points in time as one or more instances (e.g., values or artifacts).

Content is interpretable by references to a
```
Now spending one paragraph on each concern, reintroducing revised versions of the summaries we had in the RAM 3.0.

shared, formally defined Concept, whereas links to a particular

> *C*ontext (in terms of time, place, or real-world entities) make the
> content potentially relevant for certain Data Consumers. So the upper

Reviewer: ```suggestion
*C*ontext (in terms of, e.g.,
* time, 
* place, or 
* real-world entities)
make the
Content potentially relevant for certain Data Consumers.

Thus, the upper
```
Now spending one paragraph on each concern, reintroducing revised versions of the summaries we had in the RAM 3.0.

part of the C-Hexagon deals with the what aspects, independently of
Data Exchange, Data Sharing or Data Utilization. The lower part relates
to the how aspects; i.e. how the content is exchanged

> (*C*ommunication) and under which conditions (*C*ommodity). The

Reviewer: ```suggestion
(*C*ommunication) and under which conditions (*C*ommodity). 

The *Communication* concern deals with means to communicate a Resource's Content in one of the Representations
available, e.g., 
* by sending messages in some communication protocol
* to a resource or service endpoint or to an IDS Connector
* in order to perform an operation.

The *Commodity* concern helps to address the value and utility of a Resource in terms of, e.g.,
* its provenance,
* its quality, and
* the (usage) policies attached to it, e.g., the obligation to pay a certain price for its consumption.

The 
```
Now spending one paragraph on each concern, reintroducing revised versions of the summaries we had in the RAM 3.0.

*C*ommunity of Trust concern refers to the distinctive feature of the
International Data Spaces being an ecosystem of certified participants

> and components that exchange and share Digital Resources in accordance
> with usage policies ensuring data sovereignty.

Reviewer: ```suggestion
operating certified components, such as Connectors. Using such components, Participants exchange and share Digital Resources in a secure and trusted way in accordance
with contracts composed of usage policies, thus ensuring data sovereignty.
```
Now spending one paragraph on each concern, reintroducing revised versions of the summaries we had in the RAM 3.0.

> 
> The level of detail differs across the individual concerns. The
> selection of their constituting aspects may change in light of new
> requirements and insights. Modeling concerns may inform, but do not

Reviewer: ```suggestion
::: Center
![Detailed Concern Hexagon](./media/image53.png){#fig:Detailed_Concern_Hexagon}
:::

The level of detail differs across the individual concerns. The
selection of their constituting aspects may change in light of new
requirements and insights; Fig. [3](#fig:Detailed_Concern_Hexagon){reference-type="ref"
reference="fig:Detailed_Concern_Hexagon"} suggests one such expansion of the C-Hexagon to one more level of detail.

Modeling concerns may inform, but do not
```
Reinstated the old detailed C-Hexagon figure, now clarifying that this it no gospel but one possible approach.

necessarily correspond to any physical organization of the model (e.g.,
modules or directories). Some of the models listed below directly map to
the above mentioned concerns, while others take a more detailed
perspective on particular aspects.

[^1]: IDSA members may find further information at
    https://industrialdataspace.jiveon.com/community/workinggroups/architecture/swg4-information-model/.

[^2]: https://www.odata.org/

[^3]: https://opcfoundation.org/

[^4]: https://github.com/International-Data-Spaces-Association/InformationModel

[^5]: https://www.w3.org/standards/semanticweb/

[^6]: https://www.w3.org/TR/vocab-dcat-2/

[^7]: https://www.w3.org/TR/odrl-model/

[^8]: https://www.w3.org/TR/skos-reference/

> 

Reviewer: ```suggestion

[^rdfs]: RDF Schema 1.1. W3C Recommendation 25 February 2014. https://www.w3.org/TR/rdf-schema/

[^owl]: OWL 2 Web Ontology Language Document Overview (Second Edition). W3C Recommendation 11 December 2012. https://www.w3.org/TR/owl2-overview/

[^shacl]: Shapes Constraint Language (SHACL). W3C Recommendation 20 July 2017. https://www.w3.org/TR/shacl/

[^sparql]: SPARQL 1.1 Query Language. W3C Recommendation 21 March 2013. https://www.w3.org/TR/sparql11-query/

```
References to the basic W3C standards that we now mention above

> 

Reviewer: ```suggestion

[^bp-ld]: Best Practices for Publishing Linked Data. W3C Working Group Note 09 January 2014. http://www.w3.org/TR/ld-bp/

```
References to the basic W3C standards that we now mention above
